js/index.js

```diff
@@ -1 +1,189 @@
-// JS goes here
\ No newline at end of file
+// JS goes here
+
+//Upon click, which the hamburger image to 'close' and change nav display: none => flex;
+
+let hamburger = document.querySelector('.nav-ham');
+let nav = document.querySelector('nav');
+let headNav = document.querySelector('.nav');
+
+hamburger.addEventListener('click', fromHam, {once: true});
```

Comment:
interesting pattern! for updating the DOM. Another one could be to make it a component with a open and close property. Or if you use CSS `class` to toggle the styles, you could check for that